rdkit/Chem/UnitTestDescriptors.py

```diff
@@ -113,11 +119,31 @@ def testMQN(self):
          292, 41, 20, 1852, 5642, 31, 9, 1, 2, 3060, 1750])
     fn = os.path.join(RDConfig.RDCodeDir, 'Chem', 'test_data', 'aromat_regress.txt')
     ms = [x for x in Chem.SmilesMolSupplier(fn, delimiter='\t')]
-    vs = np.zeros((42, ), np.int32)
+    vs = np.zeros((42,), np.int32)
     for m in ms:
       vs += rdMolDescriptors.MQNs_(m)
     self.assertFalse(False in (vs == tgt))
 
-# - - - - -
+  def test_FpDensityMorgan(self):
+    m = Chem.MolFromSmiles('C')
+    self.assertEqual(Descriptors.FpDensityMorgan2(m), 1)
+    m = Chem.MolFromSmiles('CC')
+    self.assertEqual(Descriptors.FpDensityMorgan2(m), 1)
+    m = Chem.MolFromSmiles('CCC')
+    self.assertEqual(Descriptors.FpDensityMorgan2(m), 4.0 / 3)
```

Comment:
I'm happy with the version changes.  I think we might be able to expect floating point identity here, but I've been surprised before.